pysubtools/parsers/base.py

```diff
@@ -68,8 +67,8 @@ def __init__(self, stop_level='error'):
         self._current_line = None
 
     def _add_msg(self, level, line_number, column, line, description):
-        if (self._stop_level and self.LEVELS.index(level) >=
-                self.LEVELS.index(self._stop_level)):
+        if (self._stop_level and self.LEVELS.index(level)
```

Comment:
I think it would be more readable if breaking at `and`.

We have the first part of condition to safeguard so we don't use undefined `_stop_level` in the second part of the criteria. Other way how to solve this is, to ensure `_stop_level` always have a value, so you get a simpler condition.

Also, if you don't understand the condition at the beginning, try to understand it and give it a name. If you can, put it into a variable with that name. For this case, it could be named `is_suppressed_message` and use it as `if not is_suppressed_message`. Of course, you need to flip to condition for this.

Just some ideas for improving legacy code :).